intro2.py

```diff
@@ -0,0 +1,15 @@
+# Write a script that checks if a given number is palindrome (eg. 2, 33, 13231). Do not use strings.
+
+given_number = int(raw_input('Please enter a number:'))
+number = given_number
+contsructed_number = 0
+
+while number > 0:
+    digit = number % 10
+    contsructed_number = contsructed_number *10 + digit
+    number = number / 10;
```

Comment:
Same thing here with the differences between python versions. This returns a float in python 3, so the algorithm will fail. http://sebastianraschka.com/Articles/2014_python_2_3_key_diff.html#integer-division